lib/stack_master/stack_definition.rb

```diff
@@ -7,15 +7,29 @@ class StackDefinition
       attribute :stack_name, String
       attribute :template, String
       attribute :tags, Hash
+      attribute :s3, Hash
       attribute :notification_arns, Array[String]
       attribute :base_dir, String
       attribute :secret_file, String
+      attribute :files, Array[String]
       attribute :stack_policy_file, String
       attribute :additional_parameter_lookup_dirs, Array[String]
     end
 
+    def template_dir
+      File.join(base_dir, 'templates')
+    end
+
     def template_file_path
-      File.join(base_dir, 'templates', template)
+      File.join(template_dir, template)
+    end
+
+    def files_to_upload
+      require 'pry'
+      binding.pry
```

Comment:
The debug lines shouldn't be in the PR. Is this method used anywhere? Should we be calling it from https://github.com/envato/stack_master/pull/87/files#diff-cf22d74c8f612a3282d758a02c9d21e7R116 ?